tutor_webhook_receiver/plugin.py

```diff
@@ -2,14 +2,44 @@
 import os
 import pkg_resources
 
+from .__about__ import __version__
 
 templates = pkg_resources.resource_filename(
     "tutor_webhook_receiver", "templates"
 )
 
-config = {}
+config = {
+    "defaults": {
+        "VERSION": __version__,
+        "DOCKER_IMAGE": "webhook_receiver/webhook_receiver:latest",
+        "HOST": "webhooks.{{ LMS_HOST }}",
+        "DB_ENGINE": "django.db.backends.mysql",
+        "DB_NAME": "webhook_receiver",
+        "DB_USER": "webhook_receiver01",
+        "DB_PASSWORD": "",
+        "DB_HOST": "{{ MYSQL_HOST }}",
+        "DB_PORT": "{{ MYSQL_PORT }}",
+        "DB_MIGRATION_OPTIONS": {},
+        "DJANGO_SECRET_KEY": "",
```

Comment:
This secret key would also need to be autogenerated, at least on `quickstart`. I am not entirely sure if the plugin API has the ability to hook into the `quickstart` mechanism; please find out. :)

Reply:
I think what you want here is `config["add"]`, as documented [here](https://docs.tutor.overhang.io/plugins/api.html#config).

Reply:
Yes will do